Userland/Libraries/LibWeb/Layout/ListItemMarkerBox.cpp

```diff
@@ -36,6 +36,12 @@ ListItemMarkerBox::ListItemMarkerBox(DOM::Document& document, CSS::ListStyleType
     case CSS::ListStyleType::UpperLatin:
         m_text = String::bijective_base_from(m_index - 1);
         break;
+    case CSS::ListStyleType::LowerRoman:
+        m_text = String::roman_number_from(m_index).to_lowercase();
```

Comment:
It is a bit unfortunate that we have to allocate two `String` objects here, one for the uppercase and one for the lowercase variation. Since this is not performance critical code, and I do not see a simple way to avoid this, I think this is fine though.